io/genbank/genbank.go

```diff
@@ -35,6 +38,9 @@ func Parse(file []byte) poly.Sequence {
 	// Create sequence struct
 	sequence := poly.Sequence{}
 
+	// Add the SHA256 hash to sequence.SHA256
+	sequence.MD5 = md5.Sum(file)
```

Comment:
Does Genbank publish their file hashes and how do we check against them?

Reply:
No. This is for if you want to check if a genbank file has changed.